src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(calculateStatistics(fromFileName), toFileName);
+
+    }
+
+    public int [] calculateStatistics(String fromFileName) {
+        File file = new File(fromFileName);
+        int [] result = new int[3];
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String splitInput = reader.readLine();
+
+            while (splitInput != null) {
+                String [] temp = new String[2];
+                temp = splitInput.split(",");
```

Comment:
There is no need to initialize `temp` with a new `String[2]` before immediately reassigning it to the result of `splitInput.split(',')`. You can declare it without initialization: `String[] temp = splitInput.split(',');`